FrontEnd/Hopsy/src/screens/HomeScreen.js

```diff
@@ -91,6 +53,43 @@ export default class Test extends React.Component {
   }
 }
 
+posts = [{
+  id: "1",
+  name: "SLO Brew",
+  text:
+      "Polyvision concert this weekend!",
+  timestamp: Date.now(),
+  avatar: require("../images/Bunny.png"),
```

Comment:
What is the difference between avatar and image?

Reply:
The avatar is the user's profile image and the image is the post content.